sdk/src/Types.ts

```diff
@@ -104,51 +108,38 @@ export interface CrashReportPayload {
   };
 }
 
-interface TimingMessage {
-  type: 'p' | 'n';
-  duration: number;
-}
-
-interface RUMData {
-  name: string;
-  timing: TimingMessage;
-}
+//-------------------------------------------------------------------------------------------------
+// REAL USER MONITORING SPECIFIC TYPES
+//-------------------------------------------------------------------------------------------------
 
-enum RUMEventTypes {
+export enum RealUserMonitoringEvents {
   SessionStart = 'session_start',
   SessionEnd = 'session_end',
-  Timing = 'mobile_event_timing'
+  EventTiming = 'mobile_event_timing',
+  ActivityLoaded = 'p',
```

Comment:
ActivityLoaded should be renamed to ViewLoaded as Activity is only relevant to Android.